### PR TITLE
Contrib composable

### PR DESCRIPTION
**Pull request by wazolab**
Create `useContrib()` composable for more reusability

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -71,11 +71,7 @@ export default defineNuxtConfig({
     async (_options, nuxt) => {
       nuxt.hooks.hook('vite:extendConfig', config =>
         // @ts-expect-error: Do we really need to extend the config ?
-        config.plugins.push(vuetify({
-          autoImport: {
-            labs: true,
-          },
-        })))
```
Comment (frodrigo) [inline nuxt.config.ts]: You add this in a previous commit. Juste remove it from the history.

File: components/PoisCard/PoiCardContent.vue [hunk 2]
```diff
@@ -36,6 +35,16 @@ export default defineNuxtComponent({
     },
   },
 
+  data() {
```
Comment (frodrigo) [inline components/PoisCard/PoiCardContent.vue:40]: Missing typing ?

File: components/PoisList/PoisTable.vue [hunk 3]
```diff
@@ -25,76 +21,72 @@ export default {
...
+    context(): PropertyTranslationsContextEnum {
+      return PropertyTranslationsContextEnum.List
     },
   },
-}
+})
 </script>
 
 <template>
-  <v-data-table
```
Comment (frodrigo) [inline components/PoisList/PoisTable.vue]: Changes not linked with contrib composable ?

**Comment by wazolab** [comment]
We should be ok now, some commits came along during changes extraction ... sorry :/

**Pull request merged**